### ISSUE TITLE
Additional Profile Data in Table(s) on the Site

### ISSUE THREAD

**Issue by MrPliffles** [emperorshammer/issues-incoming-queue#229]
There's almost certainly a better way to do this, but I'm using https://tc.emperorshammer.org/stats.php?type=flightcertification&rank=all to pull all the FCHG point values into Excel and I like that setup because I can have Excel pull the data from the table via "Refresh All" and it's great, no need to change that setup or build a replacement.

What I could use is more data on tables like that or, even better, a new table that gives me each pilot's name (by itself, no rank), their rank, their PIN, their assigned squadron, and their assigned squadron's ID number (e.g., Alpha is squadron 7) in their own columns. That'd let me update some roster information that I've been building manually when I go FCHG and combat rating award updates.

**Comment by turtlexiv** [comment]
Easy enough. I could just give it to you as a CSV if you want.

**Comment by MrPliffles** [comment]
Yeah, that'd be great. Doesn't bother me if it's not a permanent function on the site as long as I can get the data.

**Comment by turtlexiv** [comment]
How does this look?

![image](https://github.com/user-attachments/assets/811f3017-cf48-413f-9690-71aaafd34a88)

**Comment by MrPliffles** [comment]
Looks great!
